jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -471,6 +471,56 @@
               echo "Exiting with code: ${{rc}}"
               exit ${{rc}}
             fi
+      - build-test-kubemark-e2e-gce: # kubernetes-pull-build-test-kubemark-e2e-gce
+          status-context: Kubemark GCE e2e
+          max-total: 12
+          trigger-phrase: 'kubemark\s+test'
+          cmd: |
+            export KUBE_SKIP_PUSH_GCS=y
+            export KUBE_RUN_FROM_OUTPUT=y
+            export KUBE_FASTBUILD=true
+            ./hack/jenkins/build.sh
+
+            # Nothing should want Jenkins $HOME
+            export HOME=${{WORKSPACE}}
+
+            export KUBERNETES_PROVIDER="gce"
+            export E2E_NAME="e2e-kubemark-${{NODE_NAME}}-${{EXECUTOR_NUMBER}}"
```

Comment:
may want to make sure this name is short enough - it's used in a number of resources which have length limits for names.